.github/workflows/test.yml

```diff
@@ -43,19 +43,9 @@ jobs:
 
   test-submodules:
     runs-on: ubuntu-latest
-    container: tendermintdev/docker-tm-db-testing
+    container: ghcr.io/faddat/archlinux/cosmos
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-go@v3
-        with:
-          go-version: 1.18
-      - uses: technote-space/get-diff-action@v6.0.1
```

Comment:
this action also caused me trouble with gaia.  in this case, it was looking for ownership of the repository.  Since we want the SDK to be forkable, this isn't desirable behavior.

Reply:
Found a solution

Reply:
what is the solution?